aws/table_aws_backup_vault.go

```diff
@@ -0,0 +1,187 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/errors"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/backup"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsBackupVault(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_backup_vault",
+		Description: "AWS Backup Vault",
+		// Get: &plugin.GetConfig{
+		// 	KeyColumns:        plugin.AnyColumn([]string{"name"}),
+		// 	ShouldIgnoreError: isNotFoundError([]string{"InvalidParameterValue"}),
+		// 	Hydrate:           getAwsBackupVaultAccessPolicy,
+		// },
+		GetMatrixItem: BuildRegionList,
+		List: &plugin.ListConfig{
+			Hydrate: listAwsBackupVaults,
+		},
+		Columns: []*plugin.Column{
```

Comment:
Use `awsRegionalColumns` for AWS common columns (i.e. account_id, partition and region)